packages/muze-axis/src/cartesian-axis/helper.js

```diff
@@ -312,15 +312,39 @@ export const calculateContinousSpace = (context) => {
     };
 };
 
+/**
+ * Overwrites domain with user defined domain (if present)
+ * @param {Object} context reference to current axes
+ * @param {Array} domain default domain
+ *
+ * @return {Array} domain
+ */
+export const getValidDomain = (context, domain) => {
+    const { domain: userDom } = context.config();
+
+    return userDom || domain;
+};
+
 export const setContinousAxisDomain = (context, domain) => {
-    const { nice, domain: userDom } = context.config();
-    if (userDom) {
-        domain = userDom;
-    }
-    if (domain.length && domain[0] === domain[1]) {
-        domain = [0, +domain[0] * 2];
+    const { nice } = context.config();
+    const scale = context.scale.bind(context);
```

Comment:
Why is bind required here?

Reply:
`scale` method is being assigned to a variable and used in the next lines for better readability. For the method to preserve it's this reference, bind has been used.